SRC/NBASIC/nbas111e.txt

```diff
@@ -464,34 +461,34 @@ byte-based and is as follows:
 
   &B11000000 = &HC0
 
-The (de)compression is performed through consecutive segments; that is, after 
+The (de)compression is performed through consecutive segments; that is, after   
 (de)compressing from/to address &H3FFF of segment S, the process continues in 
 address &H0000 of segment S+1.
 
 
 5.1 ERRORS
 
-The following error codes can be returned by the graphic compresssion and 
+The following error codes can be returned by the graphics compression and       
 decompression functions:
 
 -1: Error when compressing or decompressing. The specified segment does not
     exist, refers to VRAM or is the 255. Also, this error appears when an
     invalid VRAM address is specified in any input parameter.
- 5: Error when compressing. Segments have been exhausted before having
+ 5: Error when compressing. Segments have been depleted before having
     compressed all the required graphic data.
- 6: Error when decompressing. An invalid data is found, or segments have been
-    exhausted before finding the end of data mark.
+ 6: Error when decompressing. Invalid data found, or segments have been
+    depleted without having the 'end of data' mark met.
 
 
-6. BASIC PROGRAMS STORAGE AND EXECUTION
+6. ALLOCATING AND RUNNING BASIC PROGRAMS IN RAM 
 
-NestorBASIC includes functions that allows the storage of BASIC programs in 
-RAM or VRAM segments, and its activation or execution, without losing the 
-existing variables generated by the original program. 
+NestorBASIC includes functions that let you put BASIC programs in any segment   

```

Comment:
"store" better than "put" I think

Reply:
> The binary, generated by COMPASS 1.2, is 17K 
Is it possible at all to get 17K file assembled from NBASIC.ASM or something's wrong here?

Reply:
I believe, "put" is just fine here, just to not repeat "to store" all the time. And, probably, some "to place" here and there won't harm :)

Reply:
But the thing is that "store" is the word with the best meaning in this context. You are storing the code in a segment as you might be storing it in a disk file, so you are using memory as kind of a temporary mass storage device. "Put" and "place" are more generic words that don't convey the same meaning, you could put something to use or to display for example.

Reply:
> 

Right, agree, let it be "store" again.